plone/app/layout/globals/context.py

```diff
@@ -163,7 +169,14 @@ def folder(self):
 
     @memoize
     def is_folderish(self):
-        return bool(getattr(aq_base(aq_inner(self.context)), 'isPrincipiaFolderish', False))
+        cond1 = bool(
+            getattr(aq_base(aq_inner(self.context)),
+                    'isPrincipiaFolderish', False))
+        cond2 = False
+        if HAS_PAC:
+            if ICollection.providedBy(self.context):
+                cond2 = True
+        return cond1 or cond2
```

Comment:
No. Folderish collections have been removed in Plone 4.0/4.2: http://plone.293351.n2.nabble.com/Nested-collection-removal-in-Plone-4-0-4-2-td7556157.html